lib/shared/types/src/types/config/configBody.ts

```diff
@@ -38,7 +38,13 @@ export {
     Environment as PublicEnvironment,
 }
 
+export class Test {
+    defaultProp = true
+}
+
 export class ConfigBody<IdType = string> {
+    isConfigBody = true
```

Comment:
add a comment explaining what this is for so someone encountering a type error wouldnt just set this field manually